### PR TITLE
[UI] language status icon size

### PR DESCRIPTION
**Pull request by everhopingandwaiting**
adjust the repository language status icon size 

## before 

image

### after 

image

### CODE REVIEW

**Comment by silverwind** [comment]
10px seems to small, old one was ~18px. Suggestion:

```css
.color-icon {
  margin: .5rem 0;
  display: inline-block;
  border-radius: 100%;
  width: 14px;
  height: 14px;
  position: relative;
  top: 1px;
}

**Comment by lafriks** [comment]
Yeah, looks a bit too small

**Comment by lafriks** [comment]
14 could probably be more sensible

**Comment by everhopingandwaiting** [comment]
image

@silverwind seems a little big ， if change the height size to 14px

**Comment by silverwind** [comment]
> seems if change the height size to 14px, need to adjust the class of item , otherwise the icon can not align with text

The `position` and `top` in my example is for the vertical alignment. Flexbox centering might even be better.

**Review by 6543 — CHANGES_REQUESTED** [review]
to smal ...

**Comment by everhopingandwaiting** [comment]
image

File: web_src/less/_base.less [hunk 1]
```diff
@@ -1193,13 +1193,17 @@ i.icon.centerlock {
 }
 
 .color-icon {
-    padding-right: .7em;
-    padding-left: .5em;
+    //padding-right: .7em;
```
Comment (lafriks) [inline web_src/less/_base.less]:
```suggestion
```

File: web_src/less/_base.less [hunk 2]
```diff
@@ -1193,13 +1193,17 @@ i.icon.centerlock {
...
+    //padding-left: .5em;
```
Comment (lafriks) [inline web_src/less/_base.less]:
```suggestion
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Review by lafriks — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**